.github/workflows/nodejs.yml

```diff
@@ -41,7 +45,9 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
+      - name: Install npm@8 for Node.js 14
+        run: node -e 'process.exitCode = +process.version.startsWith("v14")' || npm install -g npm@8
```

Comment:
If it _does_ start with v14 this coasts `true` to `1` which is a failing process status code? I'm confused.